starter/src/main/java/io/javaoperatorsdk/operator/springboot/starter/DependentResourceConfigurator.java

```diff
@@ -0,0 +1,8 @@
+package io.javaoperatorsdk.operator.springboot.starter;
+
+@FunctionalInterface
+public interface DependentResourceConfigurator {
+
+  String getName();
```

Comment:
Just one note regarding to this, altevatively we could avoid inheritance, so the configuration bean would not inherit this interface, but we could put here also method `getConfiguraiton` that would result in the configuration bean. 
But no strong opinion one way or another.

Reply:
or as an atlernative we could have this just as an annotation on top of a conig bean: @DependentConfiguration(dependentName="dr name", reconcilerName="optional reconciler name")